### PR TITLE
feat(db): add postgres 14 support

### PR DESCRIPTION
**Pull request by mboskamp**
Related to CAM-14179

### CODE REVIEW

File: enterprise/content/announcement.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ __Changes in Supported Environments:__
 * Support for PostgreSQL 14
 * Support for H2 2.1 (also supported as of 7.17.3+)
 * Support for IBM Websphere Liberty 22
+* Support for PostgreSQL 14
```
Comment (koevskinikola) [inline enterprise/content/announcement.md:30]: We already mention support for PostgreSQL 14 on line 27. Do we need another one?

**Review by koevskinikola — CHANGES_REQUESTED** [review]
Maybe this PR is not needed?

File: enterprise/content/announcement.md (hunk 1 above)
Comment (mboskamp) [reply enterprise/content/announcement.md:30]: Wow, completely overlooked that. Thanks for catching it. We can ignore this PR then.

**Comment by mboskamp** [comment]
Closing this PR, since we already mention Postgres 14 in the announcement. I completely overlooked this.

**Pull request closed without merging**